plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -628,13 +628,6 @@ gs_plugin_eos_blacklist_app_for_remote_if_needed (GsPlugin *plugin,
 		NULL
 	};
 
-	/* Legacy apps that have been replaced by other versions in Flathub */
-	static const char *legacy_apps[] = {
-		"com.spotify.Client",
```

Comment:
Agree with this change; but for posterity/context - the reason these were blacklisted in the reverse (from Endless rather than Flathub) is that unlike the other apps migrated from eos-apps to flathub, we kept these ones alive in eos-apps so that OS images which had "Get App" launchers / autoinstalls would still be able to fetch them. We later decided to just deal with that regression on older images because maintaining the parallel copies of them inside our infrastructure was sufficiently painful (especially VLC, a very fragile extra-data patchwork quilt of .debs) and somewhat voided the purpose of moving them to Flathub.